rds/aurora/postgres/cluster/rds.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/acorn-io/aws/rds"
+	"github.com/acorn-io/services/aws/libs/common"
+	"github.com/aws/aws-cdk-go/awscdk/v2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awsec2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awsrds"
+	"github.com/aws/constructs-go/constructs/v10"
+	"github.com/aws/jsii-runtime-go"
+	"github.com/sirupsen/logrus"
+)
+
+var engine = awsrds.DatabaseClusterEngine_AuroraPostgres(&awsrds.AuroraPostgresClusterEngineProps{
+	Version: awsrds.AuroraPostgresEngineVersion_VER_15_3(),
+})
+
+func NewRDSStack(scope constructs.Construct, props *rds.RDSStackProps) awscdk.Stack {
+	var sprops awscdk.StackProps
+	if props != nil {
+		sprops = props.StackProps
+	}
+
+	err := common.NewConfig(props)
+	if err != nil {
+		logrus.Fatal(err)
+	}
```

Comment:
I noticed that this function is also called in `main` - does it need to be called here too?

Reply:
That was redundant indeed. Removed in the latest commit.